README.md

```diff
@@ -1,44 +1,20 @@
-ember-spreadsheet-export
-==============================================================================
+# @linc-technologies/ember-spreadsheet-export
 
 Addon that encapsulates ability to render a data set as either excel or csv.
 
-Forked from [roofstock/ember-cli-data-export](https://github.com/roofstock/ember-cli-data-export).
```

Comment:
Not keeping the original credit?

Reply:
Ah, shifted below.

Reply:
Yeah, decided as a dev it's not the first thing you care about - you wanna know compat, install and usage upfront.